src/applications/check-in/components/pages/validate/ValidateDisplay.jsx

```diff
@@ -45,7 +45,15 @@ export default function ValidateDisplay({
     },
     [setDob],
   );
-
+  const handleEnter = e => {
+    if (e.key === 'Enter') {
+      validateHandler();
+    }
+  };
+  const handleFormSubmit = useCallback(e => {
```

Comment:
I'm not sure this needs to be a useCallback as it has no dependent state.. I think just a regular function would be fine here (and avoid the lint error)

Reply:
you are right! Not sure what my angle there was.